### PR TITLE
Added few files and folders to .gitignore which should not be part of…

### PR DESCRIPTION
**Pull request by NikolayIT**
… the repository when compiling with Visual Studio 2015

### CODE REVIEW

**Comment by ArseniyShestakov** [comment]
Might be add all *.opendb and *.lib files?

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,17 @@ doc/*
 VCMI_VS11.sdf
 *.ipch
 VCMI_VS11.opensdf
+
+# Visual Studio
+*.suo
+*.user
+/AI/*/RD
```
Comment (alexvins) [inline .gitignore]: RD is a release build with debug info? OK, but where are other build types?

Comment (NikolayIT) [inline .gitignore]: When compiling in Debug configuration, no new files are created.

**Comment by DjWarmonger** [comment]
Of course these files shouldn't be a part of repository, but I don't see they ever were in first place. But as you wish.

**Comment by NikolayIT** [comment]
@DjWarmonger they wasn't in the repository. They are created by Visual Studio 2015 when I compile the source.

**Pull request merged**